src/client/api/events/sub_serialized_output.c

```diff
@@ -26,10 +26,10 @@ int event_sub_tx_msg_tagged_data(event_client_handle_t client, int *mid, char co
     printf("[%s:%d]: Tag length is invalid\n", __func__, __LINE__);
     return -1;
   }
-  // 98 is the max length for string messages/transaction/tagged-data/{tag}, tag max len is 64 bytes
-  char topic_buff[98] = {0};
+  // 100 is the max length for string messages/transaction/tagged-data/0x{tag}, tag max len is 64 bytes
+  char topic_buff[100] = {0};
```

Comment:
the maximum length is 64 bytes and the max hex-encoded-tag is 128, `BIN_TO_HEX_BYTES(TAGGED_DATA_TAG_MAX_LENGTH_BYTES)` , therefore buffer size should be 
```suggestion
  // 100 is the max length for string messages/transaction/tagged-data/0x{tag}, tag max len is 64 bytes
  char topic_buff[36+BIN_TO_HEX_BYTES(TAGGED_DATA_TAG_MAX_LENGTH_BYTES)] = {0};
```

Reply:
To keep the tagged data input similar to the restful method, now accepting `tag` in bytes and `tag_len` as a sperate parameter and tag is converted to hex encoded string. The buffer legth is also corrected.